docs/2_2_common_schema.adoc

```diff
@@ -120,8 +120,6 @@ _[Note that the definition of an encoding scheme is a prerequisite in order for
 _Furthermore, note the FMI calling interface requires that strings are encoded in UTF-8._
 _Since the XML files are also required to be encoded in UTF-8, string variables need not to be transformed when reading from the XML files in to C string variables.]._
 
-The special values `NaN`, `+INF`, `-INF` for variables values are not allowed in the FMI XML files.
-
```

Comment:
This change was discussed several times and the only conclusion I can draw from the protocols is that we do not want to change this unless we have a good use case for allowing these special values for specific XML attributes, but clearly no wholesale without explaining what the semantic for each of these values for every allowed special value would be.